src/Compilers/VisualBasic/Test/Emit/CodeGen/CodeGenOverridingAndHiding.vb

```diff
@@ -476,7 +478,11 @@ BC30308: 'Public Overrides Sub Format(i As Integer, j As Integer)' cannot overri
         End Sub
 
         <Fact()>
-        Public Sub OverloadingBasedOnOptionalParameters()
+        Public Sub OverloadingBasedOnOptionalParameters_A()
+            Dim useOpts = VisualBasicParseOptions.Default
+            If InternalSyntax.Parser.CheckFeatures(InternalSyntax.Feature.ImplicitDefaultValueOnOptionalParameter, useOpts) Then
+                Assert.True(False, $"Feature{NameOf(InternalSyntax.Feature.ImplicitDefaultValueOnOptionalParameter)} is present")
+            End If
```

Comment:
I think it would be simpler and more robust long term to explicitly create parse options targeting  LanguageVersion.VisualBasic15